modules/layers/src/geojson-layer/geojson-layer.js

```diff
@@ -294,48 +382,142 @@ export default class GeoJsonLayer extends CompositeLayer {
         layerProps.lines
       );
 
+    let pointLayerProps;
+    let pointLayerUpdateTriggers;
+    if (pointType === 'circle') {
+      pointLayerProps = {
+        stroked,
+        filled,
+        radiusUnits: sizeUnits,
+        radiusScale: sizeScale,
+        radiusMinPixels: sizeMinPixels,
+        radiusMaxPixels: sizeMaxPixels,
+        lineWidthUnits,
+        lineWidthScale,
+        lineWidthMinPixels,
+        lineWidthMaxPixels,
+
+        getFillColor: this.getSubLayerAccessor(getFillColor),
+        getLineColor: this.getSubLayerAccessor(getLineColor),
+        getRadius: this.getSubLayerAccessor(getRadius),
+        getLineWidth: this.getSubLayerAccessor(getLineWidth),
+
+        transitions: transitions && {
+          getPosition: transitions.geometry,
+          getFillColor: transitions.getFillColor,
+          getLineColor: transitions.getLineColor,
+          getRadius: transitions.getRadius,
+          getLineWidth: transitions.getLineWidth
+        }
+      };
+      pointLayerUpdateTriggers = {
+        getFillColor: updateTriggers.getFillColor,
+        getLineColor: updateTriggers.getLineColor,
+        getRadius: updateTriggers.getRadius,
+        getLineWidth: updateTriggers.getLineWidth
+      };
+    } else if (pointType === 'icon') {
+      pointLayerProps = {
+        alphaCutoff,
+        billboard,
+        iconAtlas,
+        iconMapping,
+        onIconError,
```

Comment:
I suggest we drop `alphaCutoff` and `onIconError` since they are less commonly used and can be easily specified via `_subLayerProps`

Reply:
OK, I've removed these